### PR TITLE
General Dashboard Iris Comment

### PR DESCRIPTION
**Pull request by cindyzhang12**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'
- reformatted the spacing on the general dashboard

### How to Review

[//]: # 'The order in which to review files and 
what to expect when testing locally'
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'
- check GeneralDashboard.js, GeneralDashboard.css and AnnouncementList.css

### Related PRs

[//]: # "Optional - related PRs you're waiting on
/ PRs that will conflict, etc"
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'
- may be related to some other formatting fixes I'm fixing!

### Next steps

[//]: # "What doesn't work yet, what's NOT in this 
PR/has to be done "
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Screenshots

[//]: # '#### YOUR SCREENSHOTS BELOW ########'
[//]: # '############## END ##################'

### Design Status

[//]: # 'If this is a frontend PR, what is the expected 
status of the UI in this PR (lo, mid, high- fi?'
[//]: # '####### LOFI/MIDFI/HIFI ###########'
[//]: # '############## END ##################'

CC: @dfangshuo

### CODE REVIEW

File: src/screens/shared/components/AnnouncementList.js [hunk 1]
```diff
@@ -26,11 +26,14 @@ const AnnouncementList = props => {
...
-    >
-      {list}
+    <div className={(css || '').concat(' ', 'cardsCont')}>
+      {[
+        list[list.length],
+        list[list.length - 1],
+        list[list.length - 2],
+        list[list.length - 3],
+        list[list.length - 4]
+      ]}
```
Comment (ashleynguyen27) [inline src/screens/shared/components/AnnouncementList.js:36]: is this pagination?

File: src/styles/Investments.css [hunk 2]
```diff
@@ -135,6 +135,7 @@
 .status h6 {
```
Comment (dfangshuo) [inline src/styles/Investments.css:135]: make this more specific. iin general, you should prefix all css classes here with investment

**Comment by dfangshuo** [comment]
your pr also causes the dropdown to be off - please rectify

image

in general, make all css classes here ultra specific, and the easiest way to do that is to prefix everything with the name of the component, which in this case is investment.

File: src/screens/shared/components/AnnouncementList.js (hunk 1 above)
Comment (cindyzhang12) [reply src/screens/shared/components/AnnouncementList.js:36]: oh I'm not really sure what pagination is but its to get the project news lists to show only 4 entries with most recent at the top

**Comment by dfangshuo** [comment]
Hi @cindyzhang12 i dont think you addressed my comments. Did you push

**Pull request merged**